Src/DependencyCollector/Shared/Implementation/HttpProcessing.cs

```diff
@@ -217,9 +217,10 @@ internal object OnBegin(object thisObj, bool skipIfNotNew)
                         if (webRequest.Headers[RequestResponseHeaders.CorrelationContextHeader] == null)
                         {
 #if NET45
-                            if (Activity.Current.Baggage.Any())
+                            var currentActivity = Activity.Current;
+                            if (currentActivity != null && currentActivity.Baggage.Any())
```

Comment:
it would, but I would prefer to keep current code unless you have performance objections